### DISCUSSION TITLE
Different "midnight" time

### DISCUSSION THREAD

**Discussion by zekooooo** [iSoron/uhabits#441] [Feature Requests] [answered]
Hi, first gotta say I love your app, I've been using it for a couple of months now and it's helped me a lot in life :) I love the layout and the all black theme that lets me style it like old Android.

I've got just one suggestion for you, can you add a feature to select a different time from 0:00 as the boundary between days? I work nights and it's a minor annoyance that the app switches the day in the middle of my "day", and things that I do before I go to bed always end up at 94%

**Comment by sojusnik** [comment]
See https://github.com/iSoron/uhabits/issues/50

**Comment by iSoron** [comment, marked as answer]
Thank you for the suggestion, @zekooooo. In Loop 2.0, the new day starts at 3am instead of midnight.

**Comment by ghost** [comment]
> the new day starts at 3am instead of midnight.

Hi @iSoron, has the 3am change since been reverted? My day seems to start at midnight.

**Comment by slix** [comment]
I'm also curious if the changed has been reverted. My widgets change around midnight.

**Comment by slix** [comment]
Opened #607 for this re-adding this feature.
